packages/salesforcedx-vscode-core/src/util/cliConfiguration.ts

```diff
@@ -50,21 +48,13 @@ export function disableCLITelemetry() {
   );
 }
 
-export async function isCLITelemetryAllowed(
-  projectPath: string
-): Promise<boolean> {
-  if (isCLIInstalled()) {
-    try {
-      const forceConfig = await new ForceConfigGet().getConfig(
-        projectPath,
-        SFDX_CONFIG_DISABLE_TELEMETRY
-      );
-      const disabledConfig =
-        forceConfig.get(SFDX_CONFIG_DISABLE_TELEMETRY) || '';
-      return disabledConfig !== 'true';
-    } catch (e) {
-      console.log('Error checking cli settings: ' + e);
-    }
+export async function isCLITelemetryAllowed(): Promise<boolean> {
```

Comment:
Updating this method to use `ConfigAggregator` which is much faster than running `force:config:get` command.

Reply:
@lcampos Does it provide the same functionality?

Reply:
@rcoringrato-sfdc yes, it behaves the same way as the previous version of the code